train.py

```diff
@@ -1,72 +1,86 @@
 import cv2
 import numpy
+import sys
 
-from lib.utils import get_image_paths, load_images, stack_images
-from lib.training_data import get_training_data
+from utils import get_image_paths, load_images, stack_images
+from training_data import get_training_data
 
-from lib.model import autoencoder_A
-from lib.model import autoencoder_B
-from lib.model import encoder, decoder_A, decoder_B
+from model import autoencoder_A
+from model import autoencoder_B
+from model import encoder, decoder_A, decoder_B
+from getopt import getopt

```

Comment:
I would prefer the usage of `argparse` over `getopt` as we don't really need the lower level parsing that the `getopt` library provides. It would also make it consistent with other scripts that I rewrote yesterday. Have a look at `extract.py`, `convert.py` and `lib/cli.py` if you want examples.

Reply:
+1 on that, as argparse is really conveient to also build the help at the same time